backend/api/api.go

```diff
@@ -118,6 +136,39 @@ func Setup(redmineConf cfg.RedmineConfig) *fiber.App {
 		}
 	})
 
+	app.Get("/api/user/setting/:name", func(c *fiber.Ctx) error {
```

Comment:
The endpoint could be `/api/setting/:name` as we will not be getting settings other than for a user.  The same is true for the endpoint later that takes a value.

Reply:
Yes, right. I updated the endpoint now.